src/inspector_io.cc

```diff
@@ -64,407 +58,287 @@ std::string GenerateID() {
   return uuid;
 }
 
-void HandleSyncCloseCb(uv_handle_t* handle) {
-  *static_cast<bool*>(handle->data) = true;
-}
+class RequestQueueData {
+ public:
+  using MessageQueue =
+      std::deque<std::tuple<TransportAction, int,
+                  std::unique_ptr<v8_inspector::StringBuffer>>>;
```

Comment:
Might be worth using `std::deque<Message>` for some `struct Message { … };` instead, for readability?

Reply:
Done.